src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -578,15 +601,15 @@ def create_lv(name, group, extents=None, size=None, tags=None, uuid_name=False):
             '--yes',
             '-L',
             '%s' % size,
-            '-n', name, group
+            '-n', name, group, pv
```

Comment:
this is causing all builds to fail, because it is passing `""` to `lvcreate` which is invalid. One way to solve this without adding lots of `if` conditions is to define the command like:

    cmd = ['lvcreate', '--yes', '-L', '%s' % size, '-n' , name, group]
    if pv is not '':
        cmd.append(pv)

Error from functional tests show:

    Running command: /usr/sbin/lvcreate --yes -l 100%FREE -n osd-data-a4f2824a-4af9-4998-a65b-609ae7f21281 ceph-0f8b747d-c79b-4d1e-8a31-e12b7f5bc83e 
     stderr: Physical Volume "" not found in Volume Group "ceph-0f8b747d-c79b-4d1e-8a31-e12b7f5bc83e".

Reply:
I fixed this issue, but the functional tests are still failing. I've been looking at the output for some time now, but to me it doesn't look like the problems are related to this PR. Please let me know if I missed something and they actually are (and excuse my ignorance as I get to know jenkins ;) ).